frontend/awx/access/common/DeleteRoleConfirmation.tsx

```diff
@@ -31,7 +31,13 @@ export function DeleteRoleConfirmation(props: DeleteRoleConfirmationProps) {
   return (
     <Modal
       titleIconVariant="danger"
-      title={title ? title : t(`Remove ${sourceOfRole()} Access`)}
+      title={
+        title
+          ? title
+          : typeof role.team_id !== 'undefined'
+          ? t(`Remove Team Access`)
```

Comment:
Discussing with UXD on sentence casing in actions.  Hold on merge for a bit.

Reply:
@jamestalton I updated this action title to sentence case.